please/please.py

```diff
@@ -185,24 +205,27 @@ def setup():
 
 
 @app.callback(invoke_without_command=True)
-def show(ctx: typer.Context):
-    dateNow = datetime.datetime.now()
+def show(ctx: typer.Context) -> None:
+    now = datetime.datetime.now()
     user_name = config["user_name"]
-    dateNow = datetime.datetime.now()
-
     # PRINT TASKS
     if ctx.invoked_subcommand is None:
-        center_print(rich.rule.Rule("[#FFBF00] Hello " + config["user_name"] +
-                                    "! It's " + dateNow.strftime("%d %b | %I:%M %p") + "[/]", style="#FFBF00"))
+        center_print(
+            rich.rule.Rule(
+                f"[#FFBF00] Hello {user_name}! It's {now.strftime('%d %b | %I:%M %p')}[/]",
+                style="#FFBF00",
+            )
+        )
+
         # IF THERE IS NO INVOKED COMMAND, PRINT THE TASK LIST
         # PRINT QUOTE
         quote = getquotes()
-        center_print_wrap("[#63D2FF]\"" + quote["content"] + "\"[/]", "italic")
-        center_print_wrap("[#F03A47]- " + quote['author'] + "[/]\n", "italic")
+        center_print_wrap('[#63D2FF]"' + quote["content"] + '"[/]', "italic")
+        center_print_wrap("[#F03A47]- " + quote["author"] + "[/]\n", "italic")
         print_tasks(config["tasks"])
 
 
-def main():
+def load_config() -> None:
```

Comment:
This has to be reverted. Typer requires a main() function, we cannot change this.